src/app.ts

```diff
@@ -188,6 +190,49 @@ export type ContextData = {
 
 const fedifyKv = await KnexKvStore.create(client, 'key_value');
 
+let storage = new Storage();
```

Comment:
I think all of this logic should be pulled out of `app.ts` like how we have the PubSub initialisation stuff separate. This could be part of the constructor in init method of a GCPStorageService or something?

Reply:
Done in https://github.com/TryGhost/ActivityPub/pull/533/commits/3db475b46662793f806d42d721b704b7234b9740